webapp/src/components/Select/index.js

```diff
@@ -23,29 +24,25 @@ const IncomeSelect = ({
         label={labelSelect}
         onChange={({ target }) => onChangeFunction(target.value)}
       >
-        {values[0] ? (
-          values.map(value => (
-            <MenuItem key={value} value={value}>
-              {labelSelect === t('textElectionSelect')
-                ? Number(value) + 1
-                : value === 'All'
-                ? t('AllSelectYear')
-                : value}
-            </MenuItem>
-          ))
-        ) : (
-          <MenuItem value=""></MenuItem>
-        )}
+        {values.map(value => (
+          <MenuItem key={value} value={value}>
+            {labelSelect === t('textElectionSelect')
+              ? Number(value) + 1
```

Comment:
The `ternary if condition` was created to be use as a single line, is a bad practice to use a ternary nested into another ternary

please remove the ternary nested, you can solve it by creating a function to return the correct value for the item